### PR TITLE
Public access

### PR DESCRIPTION
**Pull request by fiji-flo**

### CODE REVIEW

File: k8s/templates/public-splitter.yaml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        }
+
+        location / {
+          proxy_pass http://$target;
+        }
+      }
+    }
+
+---
+apiVersion: extensions/v1beta1
```
Comment (The-smooth-operator) [inline k8s/templates/public-splitter.yaml]: you can use the newer version: "apiVersion: apps/v1"

File: k8s/templates/public-splitter.yaml [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+      labels:
+        app: public-splitter
+    spec:
+      volumes:
+      - name: public-splitter
+        configMap:
+          name: public-splitter
+      containers:
+      - name: public-splitter
+        image: nginx
```
Comment (The-smooth-operator) [inline k8s/templates/public-splitter.yaml]: no pinning this to a specific label, will make it use the newest image available, which at certain point can auto-break the application. I recommend to use a tag here

File: k8s/templates/public-splitter.yaml (hunk 1 above)
Comment (fiji-flo) [reply k8s/templates/public-splitter.yaml]: thanks

File: k8s/templates/public-splitter.yaml (hunk 2 above)
Comment (fiji-flo) [reply k8s/templates/public-splitter.yaml]: :+1:

**Pull request merged**